### DISCUSSION TITLE
Razberry - Hello world

### DISCUSSION THREAD

**Discussion by esteban-uo** [zwave-js/zwave-js#2072] [Q&A] [answered]
Hi all,
First of all, I would like to thank a lot this awesome project!

I'm trying to test this project together with a Razberry (I think 2016) (which I can use fine through the zwave.me smart home / API )

I'm facing the issue just at the beginning interview:

```
21:46:52.496 DRIVER   starting driver...
21:46:53.359 DRIVER   opening serial port /dev/ttyAMA0
21:46:53.821 DRIVER   serial port opened
21:46:53.845 SERIAL » [NAK]                                                                   (0x15)
21:46:55.482 DRIVER   loading configuration...
21:47:21.843 DRIVER   beginning interview...
21:47:21.864 DRIVER   added request handler for AddNodeToNetwork (0x4a)...
                      1 registered
21:47:21.872 DRIVER   added request handler for RemoveNodeFromNetwork (0x4b)...
                      1 registered
21:47:21.878 DRIVER   added request handler for ReplaceFailedNode (0x63)...
                      1 registered
21:47:21.899 CNTRLR   beginning interview...
21:47:21.905 CNTRLR   querying version info...
21:47:25.322 SERIAL » 0x01030015e9                                                         (5 bytes)
21:47:25.363 DRIVER » [REQ] [GetControllerVersion]
21:47:26.455 CNTRLR   Failed to execute controller command after 1/3 attempts. Scheduling next try i
                      n 100 ms.
21:47:26.579 SERIAL » 0x01030015e9                                                         (5 bytes)
21:47:26.591 DRIVER » [REQ] [GetControllerVersion]
21:47:27.634 CNTRLR   Failed to execute controller command after 2/3 attempts. Scheduling next try i
                      n 1100 ms.
21:47:28.753 SERIAL » 0x01030015e9                                                         (5 bytes)
21:47:28.764 DRIVER » [REQ] [GetControllerVersion]
21:47:29.915 DRIVER   Failed to initialize the driver: Timeout while waiting for an ACK from the con
                      troller
21:47:30.020 DRIVER   destroying driver instance...
```
Any hints on what could be wrong, will be really appreciate it. Thanks!!

**Comment by AlCalzone** [comment, marked as answer]
I'd assume that either the path is wrong or the serialport blocked by another application that uses it. You can't have multiple drivers access the controller stick/module at the same time.

**Reply by esteban-uo** [reply]
thanks a lot for your reply @AlCalzone !

Indeed the serial port was being used by the zwave default server. Just for others facing a similar issue, you can stop it with `/etc/init.d/z-way-server stop`

However, after the interview is completed it raises an exception: `Failed to initialize the driver: undefined is not a function`. I'm attaching the logs. [logs.txt](https://github.com/zwave-js/node-zwave-js/files/6140926/logs.txt)

Thanks!

**Reply by AlCalzone** [reply]
Since this has turned into a bug report, let's continue here please:
https://github.com/zwave-js/node-zwave-js/issues/2077

**Reply by esteban-uo** [reply]
it turned out I wasn't using properly the handlers after the nodes were ready. thanks a lot @AlCalzone for your very fast support!
